ast/src/analyzed/mod.rs

```diff
@@ -200,6 +206,21 @@ impl<T> Analyzed<T> {
             .sum()
     }
 
+    fn stage_declaration_type_count(&self, poly_type: PolynomialType, stage: Option<u32>) -> usize {
+        self.definitions
+            .iter()
```

Comment:
```suggestion
            .values()
```
the name is never used